hack/make-rules/test.sh

```diff
@@ -158,11 +153,10 @@ for arg; do
   fi
 done
 if [[ ${#testcases[@]} -eq 0 ]]; then
-  normalize="false"
-  kube::util::read-array testcases < <(kube::test::find_dirs)
-fi
-
-if [[ "${normalize}" == "true" ]]; then
+  # If the user passed no targets in, we want ~everything.
+  kube::util::read-array testcases < <(kube::test::find_go_packages)
+else
+  # If the user passed targets, we should normalize them.
```

Comment:
`make test-integration` still normalizes, but that's okay because it's fewer directories and fast enough. I had previously added an env variable that also covered `make test-integration`, but that was overkill.

Reply:
To fix integration, we;d need to allow the caller to say "don't normalize", right?

Reply:
Yes, that's what I had done. But I don't think it's worth the extra complexity.

Reply:
we could scan the arguments and if they taste like a "hostname/path", skip normalization.